moveit_ros/visualization/src/moveit_ros_visualization/moveitjoy_module.py

```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/env python
```

Comment:
Does this impact the ability to run this with python3 or would you need to define python3 as your system python for that to work?

Reply:
This is a module file without a main, so [it does not need the shebang](https://stackoverflow.com/questions/26650427/should-i-put-the-shebang-line-in-every-python-file).

I don't think this breaks anything w.r.t. python3, it's just general (debatable) cleanup.
@rhaschke It would have been nice to see a comment indicating that in the commit message.